tests/integ/modin/series/test_equals.py

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright (c) 2012-2024 Snowflake Computing Inc. All rights reserved.
+#
+
+import modin.pandas as pd
+import pytest
+
+import snowflake.snowpark.modin.plugin  # noqa: F401
+from tests.integ.modin.sql_counter import sql_count_checker
+
+
+@pytest.mark.parametrize(
+    "lhs, rhs, expected",
+    [
+        ([1, 2, 3], [1, 2, 3], True),
+        ([1, 2, None], [1, 2, None], True),  # nulls are considered equal
+        ([1, 2, 3], [1.0, 2.0, 3.0], False),  # float and integer types are not equal
+        ([1, 2, 3], ["1", "2", "3"], False),  # integer and string types are not equal
+    ],
+)
+@sql_count_checker(query_count=7, join_count=3)
+def test_equals_dataframe(lhs, rhs, expected):
```

Comment:
should be series? comment about constructing native object first also applies

Reply:
It's a duplicate test. removed.